css/main.css

```diff
@@ -78,3 +78,59 @@ h1 {
 .main_button:hover {
   background: #60d0d0;
 }
+
+.sect_content {
+  padding-top: 70px;
+}
+
+.sect_content .container {
+  display: flex;
+  flex-direction: row;
+}
+
+.feature {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin: 0 18px;
+}
+
+h6 {
+  font-size: 15px;
+  font-family: "Lato", sans-serif;
+  color: rgb(102, 46, 121);
+  font-weight: bold;
+  letter-spacing: 3px;
+  text-transform: uppercase;
+  text-align: center;
+  margin: 18px 0;
+}
+
+.feature:nth-of-type(2n+1) h6 {
+  margin-left: 5px;
+}
+
+.feature__icon_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  width: 50px;

```

Comment:
I suppose `width` is needed just for pixel perfect, right?

Reply:
@probil, yes. the images aren't centered in relation to the text

Reply:
@probil, should I centerise all icons in relation to text?

Reply:
@probil, done